addon/components/nypr-story/header-content.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import computed from 'ember-computed';
+import layout from '../../templates/components/nypr-story/header-content';
+
+export default Ember.Component.extend({
+  layout,
+  pk: computed('story.id', function() {
+    // coerce to string b/c ember-data expects
+    // a string for IDs
+    return String(get(this, 'story.id'));
```

Comment:
you may not need this now that the story model is a true ember-data object (but I'm not positive)